include/embed.h

```diff
@@ -32,8 +32,9 @@
 #include <QString>
 
 #include "lmms_export.h"
+#ifdef PLUGIN_NAME
```

Comment:
What does this have to do with IWYU? just curious, not even trying

Reply:
`LmmsCommonMacros.h` only defines two macros: `LMMS_STR` and `LMMS_STRINGIFY`. Both are used in `embed.h` only if `PLUGIN_NAME` is defined. So the change seems sane.

My guess why I did this: I probably compiled a CPP file which include embed.h while PLUGIN_NAME was undefined - so IWYU likely told me: "You're including that header, but in this translation unit, you are not using anything from that header" - so I wrapped the `#if` around the include to make IWYU happy.